Nautilus/Patchers/PDAEncyclopediaTabPatcher.cs

```diff
@@ -0,0 +1,24 @@
+using HarmonyLib;
+using Nautilus.Handlers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UWE;
+
+namespace Nautilus.Patchers;
+internal class PDAEncyclopediaTabPatcher
+{
+    internal static void Patch(Harmony harmony)
+    {
+        harmony.PatchAll(typeof(PDAEncyclopediaTabPatcher));
+    }
+    [HarmonyPatch(typeof(uGUI_EncyclopediaTab))]
+    [HarmonyPatch(nameof(uGUI_EncyclopediaTab.Awake))]
+    [HarmonyPostfix]
+    internal static void EncyTabAwakePostfix(uGUI_EncyclopediaTab __instance)
+    {
+      ModDataBankHandler.Initialize(__instance);
```

Comment:
This should also be a config option. Not everyone wants the mods to be listed in the encyclopedia, it should definitely be toggleable

Reply:
This isn't feasible as encyclopedia entries are stored in the save file.

Reply:
Implemented.